### PR TITLE
json_query documentation.

### PR DESCRIPTION
**Pull request by Yannig**
##### ISSUE TYPE

Docs Pull Request
##### COMPONENT NAME

Documentation for json_query filter
##### ANSIBLE VERSION

```
ansible --version
ansible 2.3.0 (devel_doc_json_query 8d85699184) last updated 2016/10/29 00:08:16 (GMT +200)
  lib/ansible/modules/core: (detached HEAD de7ec946e9) last updated 2016/10/28 23:32:25 (GMT +200)
  lib/ansible/modules/extras: (detached HEAD 8bd40fb622) last updated 2016/10/28 23:32:25 (GMT +200)
  config file = /etc/ansible/ansible.cfg
```
##### SUMMARY

Add a little section to explain how to use json_query filter.

### CODE REVIEW

**Comment by bcoca** [comment]
cc @amenonsen

File: docsite/rst/playbooks_filters.rst
Comment (amenonsen) [inline docsite/rst/playbooks_filters.rst]: Should be "JSON query filter", surely?

File: docsite/rst/playbooks_filters.rst
Comment (amenonsen) [inline docsite/rst/playbooks_filters.rst]: "with complex" → "with a complex"
"extract only" → "extract"
"data within" → "data from"

File: docsite/rst/playbooks_filters.rst
Comment (amenonsen) [inline docsite/rst/playbooks_filters.rst]: "build upon" → "built upon"
I would replace the entire second sentence with: "Here are some [examples of the jmespath syntax]" (square brackets indicate link text).

File: docsite/rst/playbooks_filters.rst
Comment (amenonsen) [inline docsite/rst/playbooks_filters.rst]: "wan't" → "want"
"you can" → "You can"

File: docsite/rst/playbooks_filters.rst
Comment (amenonsen) [inline docsite/rst/playbooks_filters.rst]: Just get rid of the task names, they don't add anything to your text. Or get rid of the text and leave the names, but then the names should be written as "Display the names of all clusters" etc.

File: docsite/rst/playbooks_filters.rst
Comment (amenonsen) [inline docsite/rst/playbooks_filters.rst]: Wait, what?

**Review by amenonsen — DISMISSED** [review]
Just a few minor fixups required, then I'm happy to merge this.

File: docsite/rst/playbooks_filters.rst
Comment (filipenf) [reply docsite/rst/playbooks_filters.rst]: We "can" not "must" use a variable. Quoting here follows the same jinja/yaml rules.

Comment (Yannig) [reply docsite/rst/playbooks_filters.rst]: Well, must is maybe a little bit strong but it's easier to use a var in this case. I'll change that.

File: docsite/rst/playbooks_filters.rst
Comment (Yannig) [reply docsite/rst/playbooks_filters.rst]: ok

File: docsite/rst/playbooks_filters.rst
Comment (Yannig) [reply docsite/rst/playbooks_filters.rst]: ok

File: docsite/rst/playbooks_filters.rst
Comment (Yannig) [reply docsite/rst/playbooks_filters.rst]: ok

File: docsite/rst/playbooks_filters.rst
Comment (Yannig) [reply docsite/rst/playbooks_filters.rst]: ok

**Comment by Yannig** [comment]
@amenonsen doc updated :)

**Comment by dharmabumstead** [comment]
Please don't merge. The Ansible documentation team will merge after an edit pass and tech review. Thanks.

**Comment by dharmabumstead** [comment]
Merged with some edits. Thanks @Yannig  and @amenonsen!

**Pull request merged**
